src/parser/jobs/whm/index.tsx

```diff
@@ -74,5 +74,10 @@ export const WHITE_MAGE = new Meta({
 			Changes: () => <>Added Action Timeline structure.</>,
 			contributors: [CONTRIBUTORS.INNI],
 		},
+		{
+			date: new Date('2022-05-07'),
+			Changes: () => <>Added Afflatus Heals to the Overheal pie chart, updated Defensive Cooldowns. 6.1 now supported.</>,
```

Comment:
We generally don't bother with these for general patches and only include a changelog for the actual changes made. If there's no job changes, no need for a changelog entry. I'd prob remove or reword the entries for 6.05/6.08 and remove the note about 6.1 here. The idea is to document what changed and only what changed.

Reply:
Removed the patch update entries.